tuichain/api/views/loans.py

```diff
@@ -363,80 +386,42 @@ def get_personal_loan_requests(request):
 
 @api_view(["GET"])
 @permission_classes((IsAuthenticated,))
-def get_all_loan_requests(request):
+def get_operating_loans(request):
```

Comment:
What does "operating" mean here? Is there an existing term that we can use instead?

Reply:
Operating means that they aren't rejected or withdrawn
Means that they still are up and running basically

Reply:
What about canceled and expired loans?

Reply:
Frontend told me they didn't need canceled and expired loans, but I can add that method as well, no problem

Reply:
The trouble is that this endpoint will return them as well.